src/org/ods/orchestration/FinalizeStage.groovy

```diff
@@ -145,7 +145,8 @@ class FinalizeStage extends Stage {
             repoPushTasks << [ (repo.id): {
                 steps.dir("${steps.env.WORKSPACE}/${MROPipelineUtil.REPOS_BASE_DIR}/${repo.id}") {
                     if (project.isWorkInProgress) {
-                        git.pushRef(repo.branch)
```

Comment:
This change needs to be reverted.
If problematic, we'll need to evaluate why.

Reply:
Doing change in PR https://github.com/opendevstack/ods-jenkins-shared-library/pull/925/